FlowCrypt/Functionality/FilesManager/FilesManager.swift

```diff
@@ -89,4 +92,27 @@ class FilesManager: FilesManagerType {
             }
         }
     }
+
+    func saveLocally(file: FileType) -> URL? {
+        let documentDirectory = try? FileManager.default.url(
+            for: .documentDirectory,
+            in: .userDomainMask,
+            appropriateFor: nil,
+            create: true
+        )
+        guard let localPath = documentDirectory?.path.appending("/\(file.name)")
+        else { return nil }
+        let url = URL(fileURLWithPath: localPath)
+        try? file.data.write(to: url)
+        return url
+    }
+
+    func remove(file: FileType) {
```

Comment:
Use consistent naming to make it clearer. If above is a `saveLocally` method that is clearly related to this, then at a minimum this should be called `removeLocal` but it's still not so clear. What does local mean? In the app's own storage not accessible by other apps?